jerry-core/ecma/base/ecma-gc.cpp

```diff
@@ -352,9 +352,43 @@ ecma_gc_mark (ecma_object_t *object_p) /**< object to mark from */
             case ECMA_INTERNAL_PROPERTY_NON_INSTANTIATED_BUILT_IN_MASK_32_63: /* an integer (bit-mask) */
             case ECMA_INTERNAL_PROPERTY_REGEXP_BYTECODE:
             case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_TARGET_FUNCTION:
+            {
+              break;
+            }
+
             case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_BOUND_THIS:
```

Comment:
Please, add comments near cases, related to bound functions, about what can be contained in the property.
For example, `/* an ecma-value */` for `case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_BOUND_THIS:`.